tests/data/bed/test_ucsc_bed12tobigbed.bed

```diff
@@ -0,0 +1,15 @@
+chr7	113726364	114271760	uc003vgt.2	0	+	113726364	113726364	0	12	302,145,91,178,98,211,90,58,138,51,219,178,	0,76634,201709,340192,376764,412573,448307,452016,542230,543282,543477,545218,
+chr7	113726364	114333827	uc003vgu.3	0	+	113726364	113726364	0	20	302,91,178,90,58,75,138,51,201,178,214,105,88,84,77,102,122,70,164,3991,	0,201709,340192,448307,452016,484497,542230,543282,543495,545218,556100,558375,565893,567616,573046,573262,575755,577140,577963,603472,
```

Comment:
Can we rename this file to `tests/data/bed/test.bed12`

Reply:
I have changed the name to tests/data/bed/test.bed12